### DISCUSSION TITLE
Welcome to TeslaBMS Discussions!

### DISCUSSION THREAD

**Discussion by BobbyBleacher** [BobbyBleacher/TeslaBMS#2] [General]
Point out all my mistakes, offer ideas for improvement.

**Comment by twanek** [comment]
Hello!

Could you manage to use the ESP32 with this library? We are also trying to figure it out, but no luck so far... The only thing what is happening, that the green led on the BMS lights up for about 3 sec when I connect the cable, but no output on the serial.

Thanks!
